### PR TITLE
Rebase develop branch to Alpine 3.17

### PR DESCRIPTION
**Pull request by nemchik**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [ ] I have read the [contributing](https://github.com/linuxserver/docker-tautulli/blob/develop/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,42 +15,51 @@ ENV TAUTULLI_DOCKER=True
...
-  git clone https://github.com/Tautulli/Tautulli.git /app/tautulli && \
+  curl -o \
+    /tmp/tautulli.tar.gz -L \
+    "https://github.com/Tautulli/Tautulli/archive/${TAUTULLI_COMMIT}.tar.gz" && \
+  tar xf \
+    /tmp/tautulli.tar.gz -C \
+    /app/tautulli --strip-components=1 && \
   cd /app/tautulli && \
-  git checkout nightly && \
+  sed -i 's/^backports.zoneinfo==0.2.1$/backports.zoneinfo==0.2.1;python_version<"3.9"/' requirements.txt && \
```
Comment (nemchik) [inline Dockerfile:39]: This will not be necessary if https://github.com/Tautulli/Tautulli/pull/1926 merges.

File: Dockerfile [hunk 2]
```diff
@@ -12,42 +15,51 @@ ENV TAUTULLI_DOCKER=True
...
+  python3 -m ensurepip && \
+  pip3 install -U --no-cache-dir \
+    pip \
+    wheel && \
+  pip3 install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/alpine-3.17/ \
+    cryptography \
+    pycryptodomex \
+    pyopenssl && \
+  pip3 install -U --no-cache-dir -r requirements.txt && \
```
Comment (thespad) [inline Dockerfile]: You'll want to point this at our wheels too

Comment (nemchik) [reply Dockerfile]: Resolved in https://github.com/linuxserver/docker-tautulli/pull/114/commits/b4efc126c05106822e6d051bb4cad393eefb2afe

**Review by aptalca — APPROVED** [review]

**Review by aptalca — COMMENTED** [review]
I don't think git is needed as a runtime dep anymore, is it?

**Comment by nemchik** [comment]
> I don't think git is needed as a runtime dep anymore, is it?

The build fails without it

**Pull request merged**